controllers/indexController.js

```diff
@@ -11,26 +11,23 @@ module.exports = (connection) => {
   const controllerMethods = {};
 
   controllerMethods.render = async (req, res) => {
-    const cars = await carsModel.getAllCars(req, res);
+    const cars = await carsModel.getAllCars();
+    const allCategories = await catalogModel.getCategories();
 
     const garage = [];
 
     const options = {
       title: TITLES.MAIN,
       isHome: true,
       cars,
+      allCategories,
       garage,
     };
 
     if (req.user) {
-      const carsModificationsId =
-        await carsModel.getCarsModFromGarageWhereUserId(req.user.id);
-
-      for (const modId of carsModificationsId) {
-        const car = await carsModel.getModificationInfo(modId);
-        garage.push(car);
-      }
-      options.garage = garage.reverse();
+      const cars = await carsModel.getCarsInfoFromGarageByUser(req.user.id);
+
+      options.garage = cars.reverse();
```

Comment:
Why  is `reverse`?

Reply:
Last car that we add to garage should be first in the list of cars in the page.